rebound/librebound.py

```diff
@@ -137,16 +137,17 @@ def integrator(self):
     @integrator.setter
     def integrator(self,value):
```

Comment:
Super minor suggestion — per PEP-8 https://www.python.org/dev/peps/pep-0008/ you should have spaces after commas in arguments lists.